doc/python/LaTeX.md

```diff
@@ -35,7 +35,7 @@ jupyter:
 
 #### LaTeX Typesetting
 
-Figure titles, axis labels and annotations all accept LaTeX directives for rendering mathematical formulas and notation, when the entire label is surrounded by dollar signs `$...$`. This rendering is handled by the [MathJax library](https://www.mathjax.org/) (version 2.7.5), which must be loaded in the environment where figures are being rendered. MathJax is included by default in Jupyter-like environments, and LateX directives work with Jupyter Notebook version 6. LaTeX directives do not work with Jupyter Notebook version 7.  When embedding Plotly figures in other contexts it may be required to ensure that MathJax is separately loaded, for example via a `<script>` tag pointing to a content-delivery network (CDN).
+Figure titles, axis labels and annotations all accept LaTeX directives for rendering mathematical formulas and notation, when the entire label is surrounded by dollar signs `$...$`. This rendering is handled by the [MathJax library](https://www.mathjax.org/) (version 2.7.5), which must be loaded in the environment where figures are being rendered. MathJax is included by default in Jupyter-like environments. When embedding Plotly figures in other contexts it may be required to ensure that MathJax is separately loaded, for example via a `<script>` tag pointing to a content-delivery network (CDN).
```

Comment:
1. The latest version of MathJax is v3 which is supported by plotly.js ? Is MathJax v3 supported here too? 
2. Instead of/In addition to https://www.mathjax.org/ we may provide a link to the related version(s) here: https://www.npmjs.com/package/mathjax?activeTab=versions

Reply:
Yes, it should be the same as for Plotly.js (https://github.com/plotly/plotly.js/pull/6073/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R90) so I've updated it.